_includes/_howto/deployment_prerequisites.md

```diff
@@ -51,25 +52,42 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/compute.networkAdmin
+      --role=roles/compute.networkViewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/editor
+      --role=roles/iam.securityReviewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/iam.securityReviewer
+      --role=roles/appengine.appViewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/resourcemanager.folderAdmin
+      --role=roles/servicemanagement.quotaViewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/storage.admin
+      --role=roles/cloudsql.viewer
+      ```
+      
+      **Project level bindings**
+      ```bash
+      $ gcloud organizations add-iam-policy-binding FORSETI_PROJECT_ID \
```

Comment:
I dont think this is the right gcloud command. It should probably be

gcloud projects add-iam-policy-binding ...

https://cloud.google.com/sdk/gcloud/reference/projects/add-iam-policy-binding